README.md

```diff
@@ -155,9 +124,9 @@ The ownership voucher created for the device in the previous step needs to be im
 
```

Comment:
^ I've found the ownership voucher in `~/var/sdo/voucher.json` 

I'm not sure if the `/var` makes a difference or not though

Reply:
Hmm, i find it in `/var/sdo/voucher.json` so i'm changing the doc to that. Thx for catching this.